kuma/banners/jinja2/banners/banners.html

```diff
@@ -0,0 +1,52 @@
+{% set banners = get_banners() %}
+<div
+    id="developer_needs"
+    class="mdn-cta-container hidden"
+    aria-expanded="false"
+    aria-hidden="true"
+    data-banner="developer_needs"
+>
+    {% if banners|count > 0 %}
```

Comment:
`banners` is a QuerySet so this line will cause a `SELECT COUNT(*) FORM banners_banner`. Even though we don't need the count. 

Since this template is jinja I'd turn the QuerySet into a list with
```jinja
{% set banners = list(get_banners()) %}
```

Then you can do:

```jinja
{% if banners %}
  {% for banner in banners %}
    ...
  {% endfor %}
  <buttons>
{% endif %}
```